src/app/Plans/components/PlansTable.tsx

```diff
@@ -42,6 +44,19 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
   plans,
   migrations,
 }: IPlansTableProps) => {
+  const filterCategories: FilterCategory[] = [
```

Comment:
Per https://github.com/konveyor/virt-ui/issues/76#issuecomment-697908751 we'll need to add filter categories for all columns except number of VMs. so I think it would be Search filters for source provider / target provider (although maybe Select instead since we can query the list of available providers), and a Select filter for the plan status (logic might be tricky there... I can't remember if I made the Select filter able to have custom filter logic and not just a plain string match. if that's hard we could wait for another PR).

@vconzola should the filters for source/target provider in the plans table be search filters or select/dropdown filters?

Reply:
@mturley Not sure what you mean by search filters or select/dropdown filters, but they should be this: https://www.patternfly.org/v4/guidelines/filters#attribute-value-textbox-filters

Reply:
@vconzola what we're calling a "search filter" is the first mockup shown at that link, where the value is typed in a text box. What we're calling a "select filter" is the second mockup there, where the value is chosen from a list.

I wasn't sure which would be better since provider names are plain text, but the list of possible values is known.

Reply:
@mturley Use what you're calling a search filter.

Reply:
Got it, thanks

Reply:
@mturley, sorry I totally zapped the comment on #76.

Adding them.